core/src/test/java/org/elasticsearch/snapshots/RepositoriesIT.java

```diff
@@ -94,7 +94,8 @@ public void testRepositoryCreation() throws Exception {
         assertThat(repositoriesMetaData.repository("test-repo-2").type(), equalTo("fs"));
 
         logger.info("--> check that both repositories can be retrieved by getRepositories query");
-        GetRepositoriesResponse repositoriesResponse = client.admin().cluster().prepareGetRepositories().get();
```

Comment:
Can we still test `prepareGetRepositories()` without any parameter? Unless we test it somewhere else?